runhouse/rns/function.py

```diff
@@ -114,8 +118,8 @@ def to(
             env = env or self.env
             env = _get_env_from(env)
 
-        if self.env:
-            new_reqs = [req for req in self.env.reqs if req not in env.reqs]
+        if env:
```

Comment:
the reason this should be self.env is because in the function constructor (which supports setting the reqs/env), we append `"./"` package to the `self.env.reqs` if not included to handle syncing the local package, and additionally add it here if it's not in the `env` parameter for the `.to` function. It's not the most intuitive, and with potentially getting rid of setting env/system in the function factory this could probably be simplified -- let's discuss more live